src/core/events/bpf.rs

```diff
@@ -299,6 +299,14 @@ pub(crate) struct CommonEvent {
 
 unsafe impl Plain for CommonEvent {}
 
+impl EventFmt for CommonEvent {
+    fn event_fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        // Don't put an extra space at the beginning as this section is 1)
+        // always displayed and 2) always first.
+        write!(f, "{}", self.timestamp)
```

Comment:
To me it feels weird that Sections have to know their relative position or account for the format of others. Not that I see a cleaner way to do without it.